scripts/ci/docker-compose/base.yml

```diff
@@ -15,10 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 ---
-version: "2.2"
+version: "3.7"
```

Comment:
I need "3.7" everywhere to get "pull_policy": never (available as of May 2021 in "3.,7".